justfile

```diff
@@ -0,0 +1,29 @@
+build:
+	cargo build
+
+schema:
+  #!/usr/bin/env sh
+  for dir in contracts/*; do
+    if [ -d "$dir" ]; then
+      echo "Generating schema for $dir"
+      (cd "$dir" && cargo schema)
+    fi
```

Comment:
Like above, would delete the raw folder here, unless for some reason you need it.